default.nix

```diff
@@ -36,15 +36,15 @@ let
           hls-src = pkgs.fetchFromGitHub {
             owner  = "haskell";
             repo   = "haskell-language-server";
-            rev    = "b43a8cd78d10d77184a895293d766a0fbceb5573";
-            sha256 = "19c0k56fqpp2rr09amgravsirpxwy4l6jiwn6yd7p9a7553n4zjx";
+            rev    = "559c384e998b363aaaff6a0021e323fbf57f9e0c";
+            sha256 = "1mpmfv8d7bh2y90zia5hgw86y2n2nanlk6agxxrqgg34r821s166";
           };
 
           ghcide-src = pkgs.fetchFromGitHub {
             owner  = "haskell";
             repo   = "ghcide";
-            rev    = "bfe35e60cc5c5cf2a2bb73ca80d6b089c3ae5731";
-            sha256 = "0722pglbfwxq5vl7a616cy3c71hmfr4avmclkg3msn0srjb69c5q";
+            rev    = "v0.4.0";
```

Comment:
this doesn't match the ghcide revision used with HLS's `559c384e998b363aaaff6a0021e323fbf57f9e0c` revision.